includes/services/TemplateEngine.php

```diff
@@ -101,6 +101,12 @@ public function __construct(
         $this->twig->addGlobal('isInIframe', testUrlInIframe());
 
         // Adds Helpers
+        $this->addTwigHelper('dump', function ($var) {
+            if (isset($this->wiki->config['debug_template']) && $this->wiki->config['debug_template'] == 'yes') {
```

Comment:
we have already a `'debug' => '[yes|no]',` parameter in yeswiki, that will show php error logs, and db query, and execution time.
Is it intended to separate `debug_template` from  `debug` or would it be ok to keep everything in one config param?